lib/colorado_lottery.rb

```diff
@@ -0,0 +1,15 @@
+class ColoradoLottery
+  attr_reader :registered_contestants,
+              :winners
+              :current_contestants
+  def initialize
+    @registered_contestants = {}
+    @winners = []
+    @current_contestants = {}
+  end
+
+  def interested_and_18?(name, lotto)
```

Comment:
i'd recommend naming these arguments exactly what they are-- a `contestant` and a `game`